libs/services/auth/testing/src/fixtures/domain.fixture.ts

```diff
@@ -1,8 +1,12 @@
 import faker from 'faker'
 import { createNationalId } from '@island.is/testing/fixtures'
 import { Domain } from '@island.is/auth-api-lib'
+import { CreateApiScope } from './types'
 
-export type CreateDomain = Pick<Domain, 'name' | 'description' | 'nationalId'>
+export type CreateDomain = Pick<Domain, 'name'> &
+  Partial<Pick<Domain, 'description' | 'nationalId'>> & {
+    apiScopes?: CreateApiScope[]
+  }
```

Comment:
Can we make `CreateDomain#name` optional like before? `FixtureFactory#createDomain` already falls back to a faker slug if it is missing.